usagov/usagov-2021
# USAGOV-1946: High severity vulnerability in twig/twig

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1946

## Description
Upgraded Drupal core-recommended from 10.2.7 to 10.2.8 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
One thing to test is to do a [drush status] and confirm the core has been upgraded to 10.2.8. But another thing to really confirm is that Synk vulnerability (brought up in the ticket) goes away.

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
